src/pylhe/__init__.py

```diff
@@ -248,7 +248,7 @@ def _indent(elem, level=0):
 class LHEInit(dict):
     """Store the <init> block as dict."""
 
-    fieldnames = ["initInfo", "procInfo" "weightgroup", "LHEVersion"]
+    fieldnames = ["initInfo", "procInfo", "weightgroup", "LHEVersion"]
```

Comment:
```suggestion
```

> If we don't explicitly use
> 
> https://github.com/scikit-hep/pylhe/blob/7519967c13c97a3e2c0a8918e2191040ddb34a3e/src/pylhe/__init__.py#L251
> 
> then should we keep them at all?

Given that this line was never getting used (else we would have seen this non-list list break) I think we can just remove this.

Reply:
I had a look at the module. Most other classes have a fromstring method, which uses fieldnames. This one LHEInit doesn't. I'm no expert to understand why. Would it be useful to add such a method?

In any case, one thing the variable fieldnames is good for is documentation. Right now the docstring is minimalistic and maybe that's worth a revamp as well?

Reply:
> Most other classes have a fromstring method, which uses fieldnames. This one LHEInit doesn't. I'm no expert to understand why. Would it be useful to add such a method?

I would be in favor of adding a `fromstring` there too. It sort of exists as part of `read_lhe_init(file_or_string)` function, but that can then be mapped to use the new `fromstring`.

Reply:
I created a PR to add fromstring https://github.com/scikit-hep/pylhe/pull/283, but just like the `read_lhe_init` it does not use `fieldnames` and I don't see the point in forcing it to use it now.